prime-dotnet-webapi/Migrations/20210901033644_PharmanetTransactionLogChangesAndDefaults.cs

```diff
@@ -0,0 +1,106 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class PharmanetTransactionLogChangesAndDefaults : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
```

Comment:
I think we should also just drop the audit columns. Maybe we need a creation timestamp but that's about it.

Reply:
Done.

Reply:
@itchison Please resolve conversation if appropriate